docs/user/alerting/action-types/resilient.asciidoc

```diff
@@ -33,37 +33,21 @@ API key secret::  The authentication key secret for HTTP Basic authentication.
      apiKeySecret: tokenkeystorevalue
 --
 
-[[resilient-connector-config-properties]]
-**`config`** defines the action type specific to the configuration and contains the following properties:
+Config defines information for the action type.
 
-[cols="2*<"]
-|===
+`apiUrl`:: An address that corresponds to *URL*.
+`orgId`:: An ID that corresponds to *Organization ID*.
 
-| `apiUrl`
-| An address that corresponds to *URL*.
+Secrets defines sensitive information for the action type.
 
-| `orgId`
-| An ID that corresponds to *Organization ID*.
-
-|===
-
-**`secrets`** defines sensitive information for the action type and contains the following properties:
-
-[cols="2*<"]
-|===
-
-| `apiKeyId`
-| A string that corresponds to *API key ID*.
-
-| `apiKeySecret`
-| A string that corresponds to *API Key secret*. Should be stored in the <<creating-keystore, {kib} keystore>>.
-
-|===
+`apiKeyId`:: A string that corresponds to *API key ID*.
+`apiKeySecret`:: A string that corresponds to *API Key secret*. Should be stored in the <<creating-keystore, {kib} keystore>>.
 
+[float]
 [[resilient-action-configuration]]
 ==== Action configuration
```

Comment:
See comment under Jira section about putting the subactions in a table.  As shown currently, the subaction headings are at the same level of the "Action configuration".

Reply:
"Configuring and testing IBM Resilient" > Configure IBM Resilient.